src/core/Utils.js

```diff
@@ -182,8 +177,14 @@ function getFileTypeExtension (mimeType) {
   return mimeToExtensions[mimeType] || null
 }
 
-// returns [fileName, fileExt]
+/**
+* Takes a full filename string and returns an array of [fileName, fileExt]
+*
+* @param {string} fullFileName
+* @return {array} [fileName, fileExt]
+*/
 function getFileNameAndExtension (fullFileName) {
+  if (!fullFileName) return
```

Comment:
is this still necessary now that we check the `file.name` before calling this function? if this sometimes returns an array and sometimes returns `undefined`, we may have to check the result type whenever we use it.

Reply:
Right, gonna change that.